### ISSUE TITLE
[Feature Req] Python test_competitor

### ISSUE THREAD

**Issue by AravindaDP** [usnistgov/ARIAC#194]
Will it be possible to provide a python version of test_competitor for the benefit of teams using python as primarily?

It necessary not need to come up from the organizers, may be PR from a competitor already familiar with ROS2 python would also do. (Strictly restricted to the capability on par or below the level of capability in CPP version of test_competitor. After all no need to spoon feed everything)

**Comment by jaybrecht** [comment]
We would like to include a python version of the test competitor but unfortunately the python bindings for MoveIt2 are not yet available. There is an [open pull request](https://github.com/ros-planning/moveit2/pull/1546) on the MoveIt2 repo that seems close to being merged. Once that is merged (and hopefully back-ported to galactic) we can explore a python version of the test competitor. 

In the meantime we welcome any pull requests from the community for a python based test competitor.

**Comment by AravindaDP** [comment]
Thanks for the clarification on the blocker with moveit_py. And also, that brings up another concern (Apology on bringing it here let me know if it warrants a separate discussion) Any specific reason to stick to galactic especially now that it's at EOL? Let's hope any future enhancements to moveit_py etc would still be backported anyway.

**Comment by jaybrecht** [comment]
Yeah the original plan with the port was to use humble but there is a bug with gazebo/ros2_control with the UR robots that makes them unusable in simulation. The issue is being tracked here: https://github.com/UniversalRobots/Universal_Robots_ROS2_Gazebo_Simulation/issues/19

**Comment by jaybrecht** [comment]
We are working to [add tutorials](https://ariac.readthedocs.io/en/latest/tutorials/tutorial_1.html) that show how to perform many ARIAC tasks using python (outside of controlling the robots). Hopefully this will be a good starting point for competitors who are more comfortable with python. Any feedback on these tutorials is appreciated. 

Also It seems that the moveit_py pull request has now been merged into the main branch of moveit2. Hopefully they choose to backport it. I just tried to build the package on galactic and it does not immediately work so there will be some effort. I would not rely on it being ready for this years competition. My advice for this year is to separate the system that is controlling the robots from your main system. That will allow your software to be mostly written in python and the node to control the robots to be written in C++. When [tutorial 5](https://ariac.readthedocs.io/en/latest/tutorials/tutorial_5.html) is finished this should be helpful in getting that started.
